docs/release-train-issue-template.md

```diff
@@ -0,0 +1,73 @@
+Release Akka HTTP $VERSION$
+
+<!--
+Release Train Issue Template for Akka HTTP
+
+(Liberally copied and adopted from Scala itself https://github.com/scala/scala-dev/blob/b11cd2e4a4431de7867db6b39362bea8fa6650e7/notes/releases/template.md)
+
+For every release, use the `scripts/create-release-issue.sh` to make a copy of this file named after the release, and expand the variables.
+
+Variables to be expanded in this template:
+- VERSION=???
+
+Key links:
+  - akka/akka-http milestone: https://github.com/akka/akka-http/milestone/?
+-->
+
+### Cutting the release
+
+- [ ] Check that open PRs and issues assigned to the milestone are reasonable
+- [ ] For minor or major versions, update the Change date in the LICENSE file and update the `licenses` url in the build.
+- [ ] Create a PR to add a release notes entry in docs/src/main/paradox/release-notes/. As a helper run `scripts/commits-for-release-notes.sh <last-version-tag>` which will output a list of commits grouped by submodule, and the closed issues for this milestone
+- [ ] Create a new milestone for the [next version](https://github.com/akka/akka-http/milestones)
+- [ ] Close the [$VERSION$ milestone](https://github.com/akka/akka-http/milestones?direction=asc&sort=due_date)
+- [ ] Make sure all important PRs have been merged
+- [ ] Wait until [main build finished](https://github.com/akka/akka-http/actions) after merging the latest PR
+- [ ] Update the [draft release](https://github.com/akka/akka-http/releases) with the next tag version `v$VERSION$`, title and release description. Use the `Publish release` button, which will create the tag.
+- [ ] Check that GitHub Actions release build has executed successfully (GitHub Actions will start a [CI build](https://github.com/akka/akka-http/actions) for the new tag and publish artifacts to Maven central via Sonatype)
+- [ ] Run a test against the staging repository to make sure the release went well, for example by using https://github.com/akka/akka-http-quickstart-scala.g8 and adding the sonatype staging repo with `resolvers += "Staging Repo" at "https://oss.sonatype.org/content/repositories/staging"`
```

Comment:
Staging is different from all other repositories. Worth keeping?

Reply:
I think what Raboof used to do, running the Akka gRPC test suite against the artifacts before releasing, always felt a little bit reassuring, but not so sure of the actual value of it.

Perhaps better to have such tests continuously running than to defer them to release.

Reply:
let's keep it for now